dia/antecedents2.rkt

```diff
@@ -0,0 +1,63 @@
+#lang racket
+
+(provide (rename-out [mb #%module-begin]) #%datum #%top #%top-interaction #%app)
+
+#| (Unstable) Module language for antecedents
+
+Format:
+  #lang abe/antecedents2 exported-antecedents-id [ideas-module imported-ideas-id]
```

Comment:
nit: same here, could be something like:

```
#lang abe/antecedents

(ideas ideas-module imported-ideas-id)

(export ...)
```

... but if this would complicate the implementation too much it's probably not worth it since I think we could generally just copy/paste the header on these files.

Reply:
See above.